sql/distsql/api.proto

```diff
@@ -24,13 +24,13 @@ import "cockroach/roachpb/data.proto";
 import "cockroach/roachpb/errors.proto";
 import "cockroach/sql/distsql/data.proto";
 import "cockroach/sql/distsql/processors.proto";
-import "cockroach/util/tracing/span.proto";
+import "cockroach/util/tracing/span_context_carrier.proto";
 import "gogoproto/gogo.proto";
 
 message SetupFlowRequest {
   optional roachpb.Transaction txn = 1 [(gogoproto.nullable) = false];
-  // trace, if set, holds the data for a tracing span context.
-  optional util.tracing.Span trace = 2;
+  // If set, the active span context of a snowball trace.
+  optional util.tracing.SpanContextCarrier trace = 2;
```

Comment:
I had on my todo list to rename the generic `trace` to something more helpful, like `trace_context` or `tracing_context`. Can we do that while we're here? Or since the new type is more clear maybe it doesn't matter anymore.